pkgs/development/php-packages/inotify/default.nix

```diff
@@ -0,0 +1,19 @@
+{ buildPecl
+, lib
```

Comment:
Could you please put those parameters on a single line? Just like other extensions.

Reply:
That's objectively worse when viewing diffs and doing merges.

Reply:
I agree and I suggest to wait for the point of view of other maintainers.

IMHO, if you want to change something, then align all of them at once. If we start doing things differently, it might become a mess.

Reply:
I don't see how different styles of defining input args is a mess :shrug:  
Let's not discuss these things on a case-by-case basis, as soon as there's agreement on a formatter & code-style it'll be consistent anyways.

Reply:
Fair enough, all good for me then.

Reply:
I have to admit that "mess" is not the proper word here.

What I meant was the fact that when one want to contribute to an open-source project, the look and feel of the contribution should match what's already existing.
I notice that none of the [php-packages](https://github.com/NixOS/nixpkgs/tree/nixos-unstable/pkgs/development/php-packages/) has adopted this style of writing the parameters.

>  Let's not discuss these things on a case-by-case basis, as soon as there's agreement on a formatter & code-style it'll be consistent anyways.

Even if I agree with the comment of the author, I still believe that it would be a better option to refactor all the `php-packages` at once in a row than having a mix of different styles, which might introduce confusion for potential new contributors (_should I do like this or like that?_). This is what I wrongly named "a mess".

Reply:
nixpkgs is a big mess due to no strict enforcing of styles. So if it's not unreadable or a horrible mess. Then meh :shrug:

Reply:
It's fine, I got it. We can agree to disagree of some stuff, I'm still new in this community, I'm still learning.